zbus/src/connection/handshake.rs

```diff
@@ -988,6 +988,7 @@ impl<S: Socket> HandshakeCommon<S> {
         let line = std::str::from_utf8(line_bytes.as_slice())
             .map_err(|e| Error::Handshake(e.to_string()))?;
 
+        trace!("Reading {line}");
```

Comment:
* Can you append "in handshake code" in the commit log for it to be more specific?
* Just FYI, Lennart was surprised that we don't optimize the handshake by sending over all lines at once, predicting the server's response. I am hoping to look into that soon.

Reply:
ok